code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -386,7 +386,7 @@
 /obj/effect/proc_holder/spell/aoe_turf/fire_cascade/proc/fire_cascade(atom/centre,max_range)
 	playsound(get_turf(centre), 'sound/items/welder.ogg', 75, TRUE)
 	var/_range = 1
-	for(var/i = 0, i <= max_range,i++)
+	for(var/i in 0 to max_range)
```

Comment:
you could change this to `for(var/range in 1 to max_range + 1)` and remove the `_range` var and have it be logically equivalent

Reply:
1b08f2a